### PR TITLE
KOGITO-772 - Upgrade Quarkus to version 1.1.0

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/kogito-bom/pull/78
- https://github.com/kiegroup/kogito-runtimes/pull/266

### CODE REVIEW

**Review by mswiderski — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
     <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
     <version.kubernetes-client>4.4.2</version.kubernetes-client>
-    <version.okhttp>3.12.0</version.okhttp>
+    <version.okhttp>3.12.6</version.okhttp>
```
Comment (ricardozanini) [inline pom.xml:61]: @cristianonicolai we might need too revert back this one or upgrade the `kubernetes-client` version also:
https://github.com/fabric8io/kubernetes-client/blob/v4.4.2/pom.xml#L85

But this `kubernetes-client` must follow Quarkus one.

Comment (cristianonicolai) [reply pom.xml:61]: @ricardozanini done :)

**Review by ricardozanini — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -57,8 +57,8 @@
     <version.org.eclipse.jdt>3.19.0</version.org.eclipse.jdt>
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
     <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
-    <version.kubernetes-client>4.4.2</version.kubernetes-client>
-    <version.okhttp>3.12.0</version.okhttp>
+    <version.kubernetes-client>4.6.4</version.kubernetes-client>
```
Comment (radtriste) [inline pom.xml]: updating kubernetes-client to 4.6.4 requires an update of jackson
see https://github.com/kiegroup/kogito-bom/pull/76
We should put version 4.6.2

**Review by nmirasch — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (cristianonicolai) [reply pom.xml]: @radtriste ack, thanks.

**Pull request merged**